amo2kinto/importer.py

```diff
@@ -28,6 +29,11 @@ def sync_records(amo_records, fields,
                  kinto_client, bucket, collection, schema, permissions):
 
     amo_records = prepare_amo_records(amo_records, fields)
+
+    if schema:
+        for record in amo_records:
+            jsonschema.validate(record, schema)
```

Comment:
Does it means I should leave it like that? Or that I should validate to_create records only?